src/main/webapp/app/shared/select/CancerTypeSelect.tsx

```diff
@@ -22,7 +18,7 @@ const getAllSubtypes = (cancerTypeList: ICancerType[]) => {
   return _.uniq(cancerTypeList.filter(cancerType => cancerType.subtype)).sort();
 };
 
-const getAllTumorTypesOptions = (cancerTypeList: ICancerType[], prevOptions) => {
+const getAllTumorTypesOptions = (cancerTypeList: ICancerType[]) => {
```

Comment:
For the naming convention, let's stick with `cancer type` instead of `tumor type`.